webapp/locales/de.json

```diff
@@ -373,6 +373,11 @@
     "questions": "Bei Fragen oder Problemen erreichst Du uns per E-Mail an",
     "title": "Human Connection befindet sich in der Wartung"
   },
+  "modals": {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-03-05T09:51:57Z" title="Thursday, March 5th 2020, 10:51:57 am +01:00">Mar 5, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -639,9 +639,12 @@
     "deleteUserAccount": {
       "accountDescription": "Sei dir bewusst, dass deine Beiträge und Kommentare für unsere Community wichtig sind. Wenn du sie trotzdem löschen möchtest, musst du sie unten markieren.",
       "accountWarning": "Dein Konto, deine Beiträge oder Kommentare kannst du nach dem Löschen WEDER VERWALTEN NOCH WIEDERHERSTELLEN!",
+      "accountWarningAdmin": "Das Konto, die Beiträge oder Kommentare können nach dem Löschen WEDER VERWALTEN NOCH WIEDERHERGESTELLT WERDEN!",
```


`VERWALTET`?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-28T13:58:42Z" title="Thursday, May 28th 2020, 3:58:42 pm +02:00">May 28, 2020</time>_

---


This is not done yet!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-26T15:08:41Z" title="Wednesday, August 26th 2020, 5:08:41 pm +02:00">Aug 26, 2020</time>_

---


Still not done yet! 🤔